### ISSUE TITLE
Inability to disable this as a "game activity"

### ISSUE THREAD

**Issue by RecklessJames** [Seklfreak/discord-image-downloader-go#44]
This is a great program! But I've noticed that it has one feature that I'm not really a fan of atm...

When in use, it shows the amount of images downloaded under my profile name... which I can't even see when I'm using the app version of Discord. I can only see this when using the browser version. Not only that, but everyone else can see it no matter what (regardless of them being on app or browser). Even more annoying, apparently its also overriding the options menu of discord. When I manually set a game or another program to display, it'll display just fine in the app version but on the browser it still shows the amount of images downloaded. Even when I have "display currently running game as a status message" completely turned off, it still bypasses this on the browser end of things lol.

In other words this program has completely taken over my ability to display programs/games anymore, or to turn it off. The reason this is an issue is because I like to have this program on to download new images to look at later, instead of me having to go through channels of content I missed.... but of course I'd also like to have it display when I'm drawing or playing certain games and such. I'd like to ask if there's any plans to address this, if its possible?

Its not the biggest of issues I'm sure, but figured I may as well bring it up... thanks for your time!

**Comment by Kakkela** [comment]
At the moment there is no ability to disable the display unless you edit the code itself and build it yourself, so in your case I'd suggest you to run downloader on alternative account.

**Comment by RecklessJames** [comment]
Well I did so and it "worked", except now its shown on _both_ accounts... even though I disconnected it from mine in the config file.  wat

**Comment by RecklessJames** [comment]
Okay thanks!
